project-base/storefront/components/Pages/Customer/OrderDetail/OrderDetailBasicInfo.tsx

```diff
@@ -32,6 +32,10 @@ export const OrderDetailBasicInfo: FC<OrderDetailBasicInfoProps> = ({ order }) =
             ),
     );
 
+    const showRepeatOrderButton = filteredOrderItems.some(
+        (item) => item.product?.isVisible === true && item.product.isSellingDenied === false,
```

Comment:
Why not write the condition as `item.product?.isVisible && !item.product.isSellingDenied`?

Reply:
I wanted to show clear signs of what is happening, but sure I can simplify it.